tensorflow/models/rnn/ptb/ptb_word_lm.py

```diff
@@ -130,8 +130,7 @@ def __init__(self, is_training, config, input_):
     #
     # The alternative version of the code below is:
     #
-    # inputs = [tf.squeeze(input_step, [1])
-    #           for input_step in tf.split(1, num_steps, inputs)]
+    # inputs = tf.unpack(inputs, num_steps, 1)
```

Comment:
I think there's an ongoing rename to move from unpack to unstack (to match numpy).  Can you make that change here?